types/rr-notifications/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for rr-notifications 1.1
+// Project: https://thegrinder.github.io/rr-notifications
+// Definitions by: Robbie Moore <https://github.com/RobbieGM>
```

Comment:
I corrected the "definitions by" field here. Could maintainers please take a look at this and approve the PR if it's ready?